docs/docsite/rst/community/collection_contributors/collection_release_with_branches.rst

```diff
@@ -0,0 +1,320 @@
+.. _collection_release_with_branches:
+
+Releasing collections with release branches
+============================================
+
+Collections MUST follow the `semantic versioning <https://semver.org/>`_ rules. See :ref:`releasing_collections` for high-level details.
+
+.. contents::
+  :local:
+
+
+Release planning and announcement
+----------------------------------
+
+#. Announce your intention to release the collection in a corresponding pinned release issue / community pinboard of the collection and in the ``#ansible-community`` `Matrix/IRC channel <https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat>`_ (and in other dedicated channels if exist) in advance.
+
+#. Ensure all the other repository maintainers are informed about the time of the following release.
+
+
+Releasing major collection versions
+-------------------------------------
+
+The new version is assumed to be ``X.0.0``.
+
+1. If you are going to release the ``community.general`` and ``community.network`` collections, create new labels ``backport-X`` and ``needs_backport_to_stable_X`` in the corresponding repositories. Copy the styles and descriptions from the corresponding existing labels.
```

Comment:
If you release the ``community.general`` and ``community.network`` collections, create new ``backport-X`` and ``needs_backport_to_stable_X`` labels in the corresponding repositories. Copy the styles and descriptions from the corresponding existing labels.

Reply:
I'll be adding these locally to my branch and pushing up the commit. Marking as 'done' as I add locally so I make sure to capture them all. Thanks!